Tests/TestCountCheckTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Xunit;
+using FluentAssertions;
+using YoCode;
+
+namespace YoCode_XUnit
+{
+    public class TestCountCheckTests
+    {
+        private readonly string testStatLine;
+        private readonly string testTestOutput;
+
+        public TestCountCheckTests()
+        {
+            testTestOutput = "Build started, please wait...\n" +
+            "Build started, please wait...\n" +
+            "Build completed.\n" +
+            "Test run for)\n" +
+            "Microsoft(R) Test Execution Command Line Tool Version 15.7.0\n" +
+            "Copyright(c) Microsoft Corporation.All rights reserved.\n" +
+            "Starting test execution, please wait...\n" +
+            "Build completed.\n" +
+            "Test run for \n" +
+            "Microsoft(R) Test Execution Command Line Tool Version 15.7.0\n" +
+            "Copyright(c) Microsoft Corporation.All rights reserved.\n" +
+            "Starting test execution, please wait...\n" +
+            "Total tests: 22.Passed: 22.Failed: 0.Skipped: 0.\n" +
+            "Test Run Successful.\n" +
+            "Test execution time: 1.6349 Seconds\n";
+            testStatLine = "Total tests: 22.Passed: 22.Failed: 0.Skipped: 0.";
+
+        }
+
+        [Fact]
+        public void Test_GetLineWithAllKeywords()
+        {
+            //testLastAuthor.ContainsAll(GitCheck.GetKeyWords()).Should().Be(true);
+            string testResult = testTestOutput.GetLineWithAllKeywords(TestCountCheck.GetTestKeyWords());
+
+            testStatLine.Should().BeEquivalentTo(testResult);
```

Comment:
For asserts like this I would follow the pattern `actual.Should().BeEquivalentTo(expected)`. It doesn't make a lot of difference here but sometimes it can, if you have something like `true.Should().Be(testResult)` or `42.3.Should().Be(testResult)`.

Reply:
👍  When you get a test failure, the failure messages make a lot more sense doing it as `actual.Should().Be(expected)`

Reply:
Also, do you really want to allow the case to be different here? `Should().Be(expected)` will check it's exactly the same, Should().BeEquivalentTo(expected)` checks that it's _almost_ the same.

The equivalent check is different for different types, but for strings it means that [the case can differ](https://github.com/fluentassertions/fluentassertions/blob/bc1285f223f9cc82aed69905891076e4d3365a73/Src/FluentAssertions/Primitives/StringAssertions.cs#L79-L80).